src/ray/protobuf/common.proto

```diff
@@ -81,6 +81,7 @@ message TaskSpec {
   bool is_direct_call = 16;
   // Number of times this task may be retried on worker failure.
   int32 max_retries = 17;
+  repeated bytes inlined_ids = 18;
```

Comment:
Instead of adding a new top level field, can we store the ids in the TaskArg proto?